README.md

```diff
@@ -95,6 +95,8 @@ txs = [
 appsSdk.sendTransactions(txs);
 ```
 
+> Note: `value` accepts a number or a string as a decimal number (hexadecimal is not supported).  
```

Comment:
Actually it is supported, we removed it, but then we brought it back because some apps already used it